mkdocs.yml

```diff
@@ -165,32 +165,32 @@ plugins:
         de: German
```

Comment:
Is this the list that is responsible for generating translated versions?
What is the criteria to include/exclude languages?

Reply:
It was: national languages and then languages with above 50% translations (= Spanish). I'm happy to use whichever criterion / algorithm you think is best :)

Reply:
:+1:  For future proofness, I'd propose we add this criteria as a comment with a link to where we can see the stats (so we realize when more languages are there ...)
I suspect autodetecting the percentage might be exploding the scope for this PR.

Reply:
And maybe let's bring it down to 30%?

Reply:
> +1 For future proofness, I'd propose we add this criteria as a comment with a link to where we can see the stats (so we realize when more languages are there ...) I suspect autodetecting the percentage might be exploding the scope for this PR.

Yes, I think it's beyond the scope of the PR. I can land another PR to do this however. Thanks to the --minimum-perc flag provided by tx, it should be difficult -- mostly a passing the languages as arguments to the mkdocs command after inspected the files pulled by tx.